maven/google-poms/requestfactory/pom-template.xml

```diff
@@ -60,7 +60,7 @@
             <dependency>
                 <groupId>com.google.code.gson</groupId>
                 <artifactId>gson</artifactId>
-                <version>2.6.2</version>
```

Comment:
Please revert the changes to the google-poms/ directory, there's no need/benefit to changing these. In fact, we probably should just remove them, as there will never be another google release...